Author: Joachim Reichel <reichel@debian.org>
Description: replace all user-visible occurrences of "normalize" by "normalize-audio"
--- a/doc/normalize-mp3.1
+++ b/doc/normalize-mp3.1
@@ -1,7 +1,7 @@
 .TH NORMALIZE-MP3 "1" "September 2001"
 .SH NAME
 normalize-mp3 - adjust levels of mp3 or ogg files by running
-normalize(1), then re-encoding
+normalize-audio(1), then re-encoding
 .SH SYNOPSIS
 \fB normalize-mp3
 [\fIOPTION\fR]... [\fIFILE\fR]...
@@ -54,8 +54,8 @@
 \fB\-V\fR
 Display version information and exit.
 .TP
-\fBThese arguments are passed as arguments to normalize.
-Run "normalize \fB\-\-help\fR" for more info.
+\fBThese arguments are passed as arguments to normalize-audio.
+Run "normalize-audio \fB\-\-help\fR" for more info.
 .TP
 \fB\-a\fR AMP
 .TP
@@ -77,7 +77,7 @@
 Report bugs to <chrisvaill@gmail.com>.
 .SH "SEE ALSO"
 The
-.B normalize(1)
+.B normalize-audio(1)
 Man page
 .SH "NOTES"
 This page was provided by Eduardo Maçan <macan@debian.org>
--- a/doc/normalize.1
+++ b/doc/normalize.1
@@ -3,22 +3,22 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "NORMALIZE" "1" "14 September 2005" "" ""
+.TH "NORMALIZE-AUDIO" "1" "14 September 2005" "" ""
 
 .SH NAME
-normalize \- adjusts volume levels of audio files.
+normalize-audio \- adjusts volume levels of audio files.
 .SH SYNOPSIS
 
-\fBnormalize\fR [ \fB\fIoptions\fB\fR ] [ \fB--\fR ] \fB\fIfile\fB\fR\fI ...\fR
+\fBnormalize-audio\fR [ \fB\fIoptions\fB\fR ] [ \fB--\fR ] \fB\fIfile\fB\fR\fI ...\fR
 
 .SH "DESCRIPTION"
 .PP
-\fBnormalize\fR is used to adjust the volume of wav audio
+\fBnormalize-audio\fR is used to adjust the volume of wav audio
 files to a standard volume level.  This is useful for things like
 creating mp3 mixes, where different recording levels on different
 albums can cause the volume to vary greatly from song to song.
 .PP
-\fBnormalize\fR operates in two phases.  In the first phase, it
+\fBnormalize-audio\fR operates in two phases.  In the first phase, it
 analyzes the specified files as wav audio files, and computes the
 volume of each file.  In the second phase, it applies a volume
 adjustment to each file to set each file's volume to a standard level.
@@ -186,7 +186,7 @@
 ID3v2 tags, you don't need it.
 .SH "CAVEATS"
 .PP
-Note that your version of \fBnormalize\fR must be
+Note that your version of \fBnormalize-audio\fR must be
 compiled with MAD library support to analyze MP3 file volume levels.
 .SH "AUTHOR"
 .PP
--- a/src/normalize-mp3.in
+++ b/src/normalize-mp3.in
@@ -61,7 +61,7 @@
   -g ADJ          |
   -n              |
   -T THR          |_ These arguments are passed as arguments to normalize.
-  -b              |  Run "normalize --help" for more info.
+  -b              |  Run "normalize-audio --help" for more info.
   -m              |
   -v              |
   -q             /
